worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java

```diff
@@ -44,10 +47,23 @@ public static boolean useVectorApi() {
         return switch (mask) {
             case SingleBlockStateMask single -> vectorizedTargetMask(single.getBlockState().getOrdinalChar());
             case InverseSingleBlockStateMask inverse -> vectorizedTargetMaskInverse(inverse.getBlockState().getOrdinalChar());
+            case ExistingBlockMask ignored -> vectorizedTargetMaskNonAir();
+            case InverseMask inverse -> {
+                final VectorizedMask base = vectorizedTargetMask(inverse.inverse());
```

Comment:
Any there any/many inverse operations that would be too intensive for this to be efficient?

Reply:
Ah yeah I see